torch/csrc/jit/fuser/kernel_spec.h

```diff
@@ -127,6 +132,7 @@ struct TORCH_API KernelSpec {
   uint64_t nInputs_;
   std::vector<std::vector<int64_t>> inputBroadcastGroups_;
   std::vector<PartitionInfo> inputChunks_;
+  std::vector<size_t> outputGradSumToSizes_;
```

Comment:
What is the `size_t`? An offset?

Reply:
Oh no. :( Something that is an index but can be -1 to say "None". So it should be int64_t probably.

Reply:
ah maybe `at::optional<size_t>` would work then... (or just `int64_t` as you're proposing)

Reply:
I prefer int64_t if I may, but if you prefer at:optional, I'd go with that.

Reply:
No preferences on my part, int64_t is fine